kubeflow/argo/argo.libsonnet

```diff
@@ -275,6 +275,8 @@
           apiGroups: [""],
           resources: [
             "configmaps",
+            "serviceaccounts",
```

Comment:
I assume this is to attach secrets and serviceaccounts to pods?

Reply:
This is related to that ksonnet serviceaccount hack. The container needs to be able to get the service account in order to generate the kubeconfig.